### PR TITLE
392 image sort bug

### LINKED ISSUE #392: Dog Images Sorting Broke
Image sorting is all goofed up.

### PR DESCRIPTION
**Pull request by docvego**
Resolves #392 

I think the problem was that the sort wasn't respecting the scope in all instances.  So there were some points where the entire Photos table was getting sorted.

Whoops :hankey:

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ gem 'therubyracer', '~> 0.11.3', platform: :ruby
 gem 'geocoder', '~> 1.1.1'
 gem 'gibbon', '~> 1.1.1'
 gem 'exception_notification', '~> 3.0.1'
-gem 'acts_as_list', '~> 0.1.8'
+gem 'acts_as_list', '~> 0.7.2'
```
Comment (TheDudeWithTheThing) [inline Gemfile]: haha, just a tiny upgrade

**Comment by TheDudeWithTheThing** [comment]
seems good but marked as in progress so not sure if you're doing more to it?

**Comment by docvego** [comment]
Nope, all done!

**Comment by TheDudeWithTheThing** [comment]
give her the push!

:+1:

**Pull request merged**